src/features/contactmoment/ContactmomentenOverzicht.vue

```diff
@@ -149,16 +149,9 @@ dt {
 
 .header-row,
 summary {
-  display: flex;
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr 2fr 1.5rem;
```

Comment:
If we chose this new implementation (which is fine by me), we don't need the `col-lg` classes anymore on the elements above.

Reply:
Another way to do this however, would be to give the final chevron item a fixed width with a margin-left auto.